packages/dds/tree/src/test/domains/json/jsonCursor.bench.ts

```diff
@@ -222,9 +223,41 @@ function extractAvgValsFromTwitter(
     cursor.exitField();
 }
 
+function extractCoordinatesFromCanadaDirect(
+    // TODO: export Canada type and use instead of any.
+    directObj: any,
+    calculate: (x: number, y: number) => void,
+): void {
+    for (const feature of directObj.features) {
+        for (const coordinates of feature.geometry.coordinates) {
+            for (const [x, y] of coordinates) {
+                calculate(x, y);
+            }
+        }
+    }
+}
+
+function extractAvgValsFromTwitterDirect(
+    directObj: TwitterJson,
+    calculate: (x: number, y: number) => void,
+): void {
+    for (const status of directObj.statuses) {
+        calculate(status.retweet_count, status.favorite_count);
+    }
+}
+
 // The original benchmark twitter.json is 466906 Bytes according to getSizeInBytes.
 const twitter = generateTwitterJsonByByteSize(isInPerformanceTestingMode ? 2500000 : 466906, true);
 describe("ITreeCursor", () => {
     bench([{ name: "canada", getJson: () => canada, dataConsumer: extractCoordinatesFromCanada }]);
     bench([{ name: "twitter", getJson: () => twitter, dataConsumer: extractAvgValsFromTwitter }]);
 });
+
+describe("Direct Object", () => {
+    jsObjectBench([
+        { name: "canada", getJson: () => canada, dataConsumer: extractCoordinatesFromCanadaDirect },
+    ]);
+    jsObjectBench([
+        { name: "twitter", getJson: () => twitter, dataConsumer: extractAvgValsFromTwitterDirect },
+    ]);
+});
```

Comment:
This can also be moved to `jsDirectObject` (which would be renamed to `jsDirectObject.bench.ts`) since it doesn't depend on anything in here.

Reply:
Moved these tests, as well as the `extractCoordinatesFromCanadaDirect` and `extractAvgValsFromTwitterDirect` functions to `jsDirectObject.bench.ts`,